xclim/indices/_agro.py

```diff
@@ -977,12 +982,13 @@ def rain_season(
 
     Notes
     -----
-    The rain season starts at the end of a period of raining (a total precipitation  of `thresh_wet_start` over `window_wet_start` days). This must
-    be directly followed by a period of `window_not_dry_start` days with no dry sequence. The dry sequence is a period of `window_dry_start`
-    days where precipitations are below `thresh_dry_start` (either the total precipitations over the period, or the daily precipitations, depending
-    on `method_dry_start`). The rain season stops when a dry sequence happens (the dry sequence is defined as in the start sequence,
-    but with parameters `window_dry_end`, `thresh_dry_end` and `method_dry_end`). The dates on which the season can start are constrained
-    by `date_min_start`and `date_max_start` (and similarly for the end of the season).
+    The rain season starts at the end of a period of raining (a total precipitation  of `thresh_wet_start` over
+    `window_wet_start` days). This must be directly followed by a period of `window_not_dry_start` days with no dry
+    sequence. The dry sequence is a period of `window_dry_start` days where precipitations are below `thresh_dry_start`
+    (either the total precipitations over the period, or the daily precipitations, depending on `method_dry_start`).
+    The rain season stops when a dry sequence happens (the dry sequence is defined as in the start sequence, but with
+    parameters `window_dry_end`, `thresh_dry_end` and `method_dry_end`). The dates on which the season can start are
+    constrained by `date_min_start`and `date_max_start` (and similarly for the end of the season).
 
```

Comment:
Ruff can remove the job of making docstring lines not too long? That's very nice.

Reply:
Ruff is more suited to letting people know when lines are too long. There are a few types of fixes considered `safe` that it can do though: https://docs.astral.sh/ruff/configuration/#fix-safety